django_scrubber/scrubbers.py

```diff
@@ -132,9 +139,9 @@ def _initialize_data(self):
 
         provider_args_str = ', '.join(str(i) for i in self.provider_args)
         provider_kwargs_str = ', '.join(str(i) for i in self.provider_kwargs)
-        logger.info('Initializing fake scrub data for provider %s(%s, %s)' %
-            (self.provider, provider_args_str, provider_kwargs_str)
-        )
+        logger.info('Initializing fake scrub data for provider %s(%s, %s)' % (
```

Comment:
Maybe use logger string formatting which may not be interpreted if the logger is disabled
```suggestion
        logger.info('Initializing fake scrub data for provider %s(%s, %s)',
```

Reply:
good point. Done!